packages/ava/src/advisor/advise-pipeline/index.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * TODO
+ * -- DataFrame 改为从 dw 直接引入
+ */
+
+import { DataFrame } from '../../data';
+import { cloneDeep } from '../utils';
+
+import { dataToAdvices } from './data-to-advices';
+
+import type { AdviseResult, ChartAdviseParams } from '../types';
+import type { BasicDataPropertyForAdvice } from '../ruler';
+
+export function advicesForChart(params: ChartAdviseParams, exportLog = false): AdviseResult {
```

Comment:
add a file advicesForChart.ts

and import the func to index

Reply:
done